pandas/tests/extension/base/interface.py

```diff
@@ -75,3 +75,22 @@ def test_copy(self, data):
 
         data[1] = data[0]
         assert result[1] != result[0]
+
+    def test_view(self, data):
+        # view with no dtype should return a shallow copy, *not* the same
+        #  object
+        assert data[1] != data[0]
+
+        result = data.view()
+        assert result is not data
+        assert type(result) == type(data)
+
+        result[1] = result[0]
+        assert data[1] == data[0]
+
+        # check that `dtype` kwarg is accepted; it is OK if it
+        #  raises NotImplementedError
+        try:
```

Comment:
use with pytest.raises

Reply:
some subclasses (e.g. DTA, PA) _dont_ raise, so we can't use pytest.raises.  all we're really checking here is that the dtype kwarg is accepted.

Reply:
then separate this to another test and override there.